### PR TITLE
adding summary info to bring into compliance

### PR DESCRIPTION
**Pull request by craigcaseyMSFT**
## Summary

For compliance, adding summaries that were missing for .NET framework 1.1

[Edit by gewarren to add preview link.]
[Sample preview link](https://review.docs.microsoft.com/en-us/dotnet/api/system.web.mobile.mobilecapabilities?view=netframework-1.1&branch=pr-en-us-4346#properties).

### CODE REVIEW

File: xml/System.Messaging/Cursor.xml [hunk 1]
```diff
@@ -105,7 +105,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Ensure that the cursor is cleared when the object is disposed.</summary>
```
Comment (gewarren) [inline xml/System.Messaging/Cursor.xml:108]:
```suggestion
        <summary>Ensures that the cursor is cleared when the object is disposed.</summary>
```

File: xml/System.Web.Mobile/MobileCapabilities.xml [hunk 2]
```diff
@@ -110,7 +110,7 @@ public void Page_Load(Object sender, EventArgs e)
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>CanCombineFormsInDeck</summary>
```
Comment (gewarren) [inline xml/System.Web.Mobile/MobileCapabilities.xml:113]: This looks like a name, not a summary.

File: xml/System.Messaging/SecurityContext.xml [hunk 3]
```diff
@@ -82,7 +82,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Ensure that the security context for a message is cleared when the object is disposed.</summary>
```
Comment (gewarren) [inline xml/System.Messaging/SecurityContext.xml:85]:
```suggestion
        <summary>Ensures that the security context for a message is cleared when the object is disposed.</summary>
```

Is this actually what the method does, or is it a note to callers?

**Comment by craigcaseyMSFT** [comment]
@gewarren  - Hi, Genevieve. Made the changes as requested, Please let me know if there's anything else that should be changed before it's good to merge. Thanks! Craig

**Comment by gewarren** [comment]
Thank you Craig!

**Pull request merged**